fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -198,14 +198,34 @@ export class Navigation implements NavigationProvider {
     NearbyFacilities.getInstance().setPpos(this.ppos);
   }
 
+  public updateRnp(rnp: number | null) {
```

Comment:
```suggestion
  public setPilotRnp(rnp: number | null) {
```
Don't want confusion around this, hence the original naming.